server/sner/server/planner/stages.py

```diff
@@ -247,7 +246,48 @@ def _run(self):
         """run"""
 
         targets = project_sixenum_targets(StorageManager.get_all_six_address(self.filternets))
-        current_app.logger.info(f'{self.__class__.__name__} projected {len(targets)} targets')
+        current_app.logger.info(f"{self.__class__.__name__} projected {len(targets)} targets")
+        self.next_stage.task(targets)
+
+
+class StorageAurorTestsslTargetlist(Schedule):  # pylint: disable=too-few-public-methods
+    """enumerates auror_testssl targets from storage data"""
+
+    def __init__(self, schedule, lockname, next_stage, ports_starttls):
+        super().__init__(schedule, lockname)
+        self.next_stage = next_stage
+        self.ports_starttls = ports_starttls
+
+    def _get_hostnames(self, host):
+        """get hostnames from host notes"""
+        hostnames = set()
+        for note in host.notes:
+            if note.xtype == "auror.hostnames" and note.data and note.data.strip():
```

Comment:
idea: do we need empty .data guards ? there should be no such in DB, or if they are we might want to break with exception so the issue is explicitly raised and cleaned instead of silently discarding the event and keeping the data ?